cypress/support/cypress-support/src/fcsSetter.js

```diff
@@ -0,0 +1,10 @@
+const setClosedCaptions = async (attribute, value) => {
```

Comment:
This file shouldn't be here.

We still want "fcsSetters" to be in the request overrides of the config module.

Also the file/module should be "fcsSetters" with an s on the end.

Reply:
Created an helper function to get from config module dynamically